lib/mayaUsd/base/tokens.h

```diff
@@ -28,9 +28,30 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 // Tokens that are used as optionVars in MayaUSD
 //
+// clang-format off
 #define MAYA_USD_OPTIONVAR_TOKENS                        \
     /* Always target a session layer on a mayaUsdProxy*/ \
-    (mayaUsd_ProxyTargetsSessionLayerOnOpen)
+    ((ProxyTargetsSessionLayerOnOpen, "mayaUsd_ProxyTargetsSessionLayerOnOpen")) \
+    /* The kind to be selected when viewport picking. */ \
+    /* After resolving the picked prim, a search from */ \
+    /* that prim up the USD namespace hierarchy will  */ \
```

Comment:
USD namespace -> USD scene ?

Reply:
Thanks @ppt-adsk, yeah, I think "namespace" is a little more precise here. See also some discussion on this in the USD object model docs here:
https://graphics.pixar.com/usd/docs/api/_usd__page__object_model.html#Usd_OM_UsdPrim